### PR TITLE
Add CAPI usages check

### PR DESCRIPTION
**Pull request by clloyd**
screen shot 2016-11-17 at 16 23 48

This adds an (unstyled) check for video usage in other content. There is a new `VideoUsages` component which makes use of a CAPI proxy to access preview CAPI (`/support/capiPreview/`). This requires a key and username for capi preview in configuration

### CODE REVIEW

File: public/video-ui/src/components/VideoUsages/VideoUsages.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  state = {
+    usages: undefined
+  }
+
+  updateUsages(videoId) {
+
+    if (!videoId) {
+      return this.setState({
+        usages: undefined
```
Comment (akash1810) [inline public/video-ui/src/components/VideoUsages/VideoUsages.js]: I did `[]` for the yt categories, shall we stick to one pattern? (happy to update the categories)

File: public/video-ui/src/components/VideoUsages/VideoUsages.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    return (
+      <li className="usages__list__item">
+        {usage}
+      </li>
+    )
+  }
+
+  renderUsages() {
+
+    const fakeUsages = [
```
Comment (akash1810) [inline public/video-ui/src/components/VideoUsages/VideoUsages.js]: so we're not talking to capi?

File: public/video-ui/src/components/VideoUsages/VideoUsages.js (hunk 1 above)
Comment (clloyd) [inline public/video-ui/src/components/VideoUsages/VideoUsages.js]: It depends on the situation, undefined in this case means we've never fetched a state, [] means we've fetched the state and there's no usages.

File: public/video-ui/src/components/VideoUsages/VideoUsages.js (hunk 2 above)
Comment (clloyd) [inline public/video-ui/src/components/VideoUsages/VideoUsages.js]: d'oh! 🍩 Will fix

**Comment by clloyd** [comment]
@akash1810 can you re-review?

File: public/video-ui/src/services/capi.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import {pandaReqwest} from './pandaReqwest';
+import {getStore} from '../util/storeAccessor';
+
+export function fetchUsages(videoId) {
+  
+  const capiProxyUrl = getStore().getState().config.capiProxyUrl;
+  return pandaReqwest({
+    url: capiProxyUrl + "/atom/media/" + videoId + "/usage",
```
Comment (akash1810) [inline public/video-ui/src/services/capi.js]: minor: could use template strings here

**Pull request merged**